art/classifiers/scikitlearn.py

```diff
@@ -770,28 +784,6 @@ def class_gradient(self, x, label=None, **kwargs):
 
         return gradients
 
-    def fit(self, x, y, **kwargs):
```

Comment:
Why were the `fit` functions removed and how is this related to the scope of this PR?

Reply:
We have looked at the removed `fit` functions during development for this PR to see if we need to update them with `check_and_transform_label_format`, and noticed that they can be made redundant and included into the `fit` function of the parent class `ScikitlearnClassifier` where we have `check_and_transform_label_format` as part of `_apply_preprocessing`.